### PR TITLE
CB-130-vehicle-schema-and-crud

### PR DESCRIPTION
**Pull request by Ry4n6137**
- [x] Vehicle schema
- [x] Vehicle CRUD

### CODE REVIEW

**Comment by Qiming-Liu** [comment]
Empty file?

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -98,46 +98,46 @@ jobs:
...
-        CLUSTER_NAME : Crankbit-cluster-${{ github.ref_name }}
-        SERVICE_NAME : crankbit_service_${{ github.ref_name }}
-        TASK_DEFINITION : crankbit_backend_task_${{ github.ref_name }}
-      run: |
-        aws ecs update-service \
-        --cluster $CLUSTER_NAME \
-        --service $SERVICE_NAME \
-        --task-definition $TASK_DEFINITION \
-        --force-new-deployment
+      - name: Checkout code
```
Comment (LH-Song) [inline .github/workflows/build.yaml:101]: why all those white space change?

File: src/controllers/vehicle.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import { Request, Response } from 'express'
+import { IVehicle } from '../models/Vehicle'
+import vehicleService from '../services/vehicleService'
+
```
Comment (LH-Song) [inline src/controllers/vehicle.ts:4]: use enum ErrorMessages {
}

Comment (LH-Song) [inline src/controllers/vehicle.ts:4]:
```suggestion
enum ErrorMessages {
}

```

Comment (LH-Song) [inline src/controllers/vehicle.ts:4]:
```suggestion
const HTTP_STATUS = {
  OK: 200,
};

const {
  OK,
} = HTTP_STATUS;

const ERROR_MESSAGES = {
  VehicleNotFound: ErrorMessages.VehicleNotFound,
};

const handleResponse = (res: Response, statusCode: number, errorMessage?: string): void => {
  if (errorMessage) {
  } else {
  }
};
```

**Comment by LH-Song** [comment]
need pass all checks first？

**Pull request closed without merging**